ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -1,458 +1,470 @@
       redistribute {{ redistribute_route }}
-{%            endfor %}
-      vlan {{ router_bgp.vlan_aware_bundles[vlan_aware_bundle].vlan }}
 {%         endfor %}
-{%     endif %}
+      vlan {{ router_bgp.vlan_aware_bundles[vlan_aware_bundle].vlan }}
+{%     endfor %}
 {# address families activation #}
 {# address family evpn activation #}
-{%     if router_bgp.address_family_evpn is defined and router_bgp.address_family_evpn is not none %}
+{%     if router_bgp.address_family_evpn is arista.avd.defined %}
    !
    address-family evpn
-{%       if router_bgp.address_family_evpn.peer_groups is defined and router_bgp.address_family_evpn.peer_groups is not none %}
-{%          for peer_group in router_bgp.address_family_evpn.peer_groups | arista.avd.natural_sort %}
-{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in is defined and router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in is not none %}
+{%         for peer_group in router_bgp.address_family_evpn.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_evpn.peer_groups[peer_group].route_map_in }} in
 {%             endif %}
-{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out is defined and router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out is not none %}
+{%             if router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out is arista.avd.defined %}
       neighbor {{ peer_group }} route-map {{ router_bgp.address_family_evpn.peer_groups[peer_group].route_map_out }} out
 {%             endif %}
 {%             if router_bgp.address_family_evpn.peer_groups[peer_group].activate == true %}
       neighbor {{ peer_group }} activate
 {%             elif router_bgp.address_family_evpn.peer_groups[peer_group].activate == false %}
       no neighbor {{ peer_group }} activate
 {%             endif %}
-{%          endfor %}
-{%       endif %}
+{%         endfor %}
 {%     endif %}
 {# address family rt-membership activation #}
-{%     if router_bgp.address_family_rtc is defined and router_bgp.address_family_rtc is not none %}
+{%     if router_bgp.address_family_rtc is arista.avd.defined %}
    !
    address-family rt-membership
-{%         if router_bgp.address_family_rtc.peer_groups is defined and router_bgp.address_family_rtc.peer_groups is not none %}
-{%            for peer_group in router_bgp.address_family_rtc.peer_groups | arista.avd.natural_sort %}
-{%               if router_bgp.address_family_rtc.peer_groups[peer_group].activate == true %}
+{%         for peer_group in router_bgp.address_family_rtc.peer_groups | arista.avd.natural_sort %}
+{%             if router_bgp.address_family_rtc.peer_groups[peer_group].activate == true %}
```

Comment:
No check if `router_bgp.address_family_rtc.peer_groups[peer_group].activate` is defined